valhalla/mjolnir/transitpbf.h

```diff
@@ -135,7 +135,7 @@ void write_pbf(const Transit& tile, const filesystem::path& transit_tile) {
   if (!filesystem::exists(transit_tile.parent_path())) {
     filesystem::create_directories(transit_tile.parent_path());
   }
-  auto size = tile.ByteSize();
+  auto size = tile.ByteSizeLong();
```

Comment:
just getting rid of a deprecated warning

Reply:
Yes, we may have to bump minimum required version of Protobuf 
Related are https://github.com/valhalla/valhalla/issues/2119 and https://github.com/valhalla/valhalla/pull/3418

Reply:
haha thanks, was about to plea for not bumping. jep, that's what I'll do, ifdef another one..

Reply:
yeah we dont have many of these, `ifdef` is a good solution